include/boost/gil/concepts/detail/type_traits.hpp

```diff
@@ -14,7 +14,7 @@ namespace boost { namespace gil { namespace detail {
 
 template <typename T>
 struct remove_const_and_reference
-    : public ::boost::remove_const<typename ::boost::remove_reference<T>::type>
```

Comment:
It's tempting to suggest to replace this use of boost type_traits by std type_traits. I take it you plan to do that in a separate PR, yes ? :-)

Reply:
Yes, in this particular case it should be possible, but I have not even looked at what trait are being used here.

A couple of attempts to get rid of Boost.TypeTraits failed and now I'm planning to take another look soon. So, yes, this will be removed in separate PR in near future :-)